### ISSUE TITLE
Use Cipheriv for counter mode of aes-256-ctr

### ISSUE THREAD

**Issue by PRFinklemeister** [chris-rock/node-encryption-examples#10]
Wen I run this sample: https://github.com/chris-rock/node-crypto-examples/blob/master/crypto-buffer.js

I get this warning: 

hello world
(node:25420) Warning: Use Cipheriv for counter mode of aes-256-ctr

Does this reduce the crypto security?

**Comment by owlstead** [comment]
Yes, it does. Actually *all the given examples* are insecure if they use [`createCipher`](https://nodejs.org/api/crypto.html#crypto_crypto_createcipher_algorithm_password_options). That method has relatively recently been deprecated but it was obviously completely insecure before that.
